FirebaseCore.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'FirebaseCore'
-  s.version          = '5.4.1'
+  s.version          = '5.4.90'
```

Comment:
Do we want to commit this to master? Or just publish a version on Staging?

Reply:
Fine to commit to master and will reset when we make the release branch. In the meantime we need to use SpecsStaging.

Reply:
Makes sense, thanks.